.github/workflows/build.yml

```diff
@@ -55,25 +55,23 @@ jobs:
     strategy:
       matrix:
         specs:
-          - { group: "models", pattern: "[a-p]*_spec.rb" }
-          - { group: "models", pattern: "[q-z]*_spec.rb" }
-          - { group: "controllers", pattern: "*_spec.rb" }
+          - { group: "models", pattern: "*_spec.rb" }
           - { group: "requests", pattern: "*_spec.rb", want_pdf: true }
-          - { group: "system", pattern: "a[a-f]*_spec.rb", want_pdf: true }
-          - { group: "system", pattern: "a[g-s]*_spec.rb", want_pdf: true }
-          - { group: "system", pattern: "a[t-z]*_spec.rb" }
+          - { group: "system", pattern: "a[a-r]*_spec.rb", want_pdf: true }
+          - { group: "system", pattern: "a[s-z]*_spec.rb", want_pdf: true }
```

Comment:
The main problem here is that almost all system tests starting with `a` are either `add`/`adding` or `assess`/`assessing`/`assessment`, so breaking it down further seems pointless. The `a[t-z]*` group had only one file in it, for example.